pixi.toml

```diff
@@ -34,19 +34,35 @@ shapely = "*"
 syrupy = "*"
 xmipy = "*"
 
-[tasks]
-# install
-install-flopy = "pip install --no-build-isolation --no-deps --disable-pip-version-check git+https://github.com/modflowpy/flopy.git"
-install-pymake = "pip install --no-build-isolation --no-deps --disable-pip-version-check git+https://github.com/modflowpy/pymake.git"
-install-modflowapi = "pip install --no-build-isolation --no-deps --disable-pip-version-check git+https://github.com/MODFLOW-USGS/modflowapi.git"
-install-modflow-devtools = "pip install --no-build-isolation --no-deps --disable-pip-version-check git+https://github.com/MODFLOW-USGS/modflow-devtools.git"
-install = { depends_on = [
-    "install-flopy",
-    "install-pymake",
-    "install-modflowapi",
-    "install-modflow-devtools",
-] }
+[pypi-dependencies]
```

Comment:
I agree it would be ideal to use `pypi-dependencies` instead of custom tasks, and originally did so in #1703, but after discussing with @jdhughes-usgs @langevin-usgs we went back to custom tasks in #1728 to get rolling CI updates for internal developer deps without needing to update the `pixi.lock` file.

Reply:
Ah I see. That's something I overlooked. I will revert back to using tasks

Reply:
I'm wondering if this is actually bad or not. The update-flopy task still works. 

The normal way of working would be:
- `pixi run install`
- some changes happen in flopy that you need
- `pixi run update-flopy`
- repeat above step when newer flopy is needed

These steps don't change when we switch to using `pypi-dependencies`

It is true that when we run the `update-flopy` we should update the lock file as well but it is not necessary.
This is because in the CI pipeline we call the  `update-flopy` task causing the locked version to lose its meaning anyway

Reply:
The name of the `update_flopy.py` script/`update-flopy` task is a bit misleading, it's not actually upgrading the flopy package, just regenerating mf6 components from DFNs.

The idea is if we use `pypi-dependencies` we just need to update the lockfile frequently as flopy/modflowapi/pymake/devtools tend to move faster than external dependencies and mf6 and flopy are often developed in sync. With a task we always get the latest in CI which is convenient for development if a little harder to debug.

Reply:
Alright. Thanks for this explanation.
I now get why the install step is required and restored that part.